.github/workflows/lint-test-build.yml

```diff
@@ -1,15 +1,12 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
 
-name: Build and Test
+name: Lint, Build and Test
 
 on:
-  push:
-    branches:
-      - '*'
   pull_request:
     branches:
-      - '*'
+      - 'main'
```

Comment:
May still be good to have this run on all pull requests? Or at least also on PR to `develop`